oscola.csl

```diff
@@ -207,7 +207,14 @@
         <text variable="title" font-style="italic" text-case="title" form="short"/>
       </if>
       <else-if type="bill legislation" match="any">
-        <text variable="title"/>
+        <choose>
+          <if variable="title-short">
```

Comment:
Habit from CSL-m coding. form="short" has a different meaning for legal types there.